README.adoc

```diff
@@ -131,6 +131,24 @@ This can be achived with the {uri-kube-apiserver}[Kubernetes API server CLI], e.
 $ kube-apiserver --cors-allowed-origins .*
 ```
 
+== Authentication
+
+We try to support the various authentication strategies supported by https://kubernetes.io/docs/reference/kubectl/overview/[kubectl] in order to provide seamless integration with your local setup. Here are the different authentication strategies we support depending on how you're using Kubebox:
+
+[cols="<,^,^,^,^,^", options="header"]
+|===
+||Executable |Server |Kubectl | Docker| Online
```

Comment:
For Docker, I wonder whether we could add the binaries to the image to make it working for GKE, EKS and DO?

Reply:
That's what I was thinking too however I'm not sure what we should do about different versions of GKE, EKS and DO. We may be lucky and they are all backwards compatible.

Also GKE, EKS and DO require some initial configuration (setting up tokens in DO, GKE for external access). We could pass them as ENV vars to docker though and set them up ourselves.

Additionally, EKS uses cmd-path which we would need to change.

Reply:
Agreed, that seems quite complex and I'm not sure that many people use the Docker client. Let's keep it that way.